### PR TITLE
[netcore] Add a few more missing SSE intrinsics.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by filipnavara** [comment]
Please add lowering for x64 or make it LLVM only. Here's a draft of how the lowering could work (without <SSE4.1 fallbacks which can be added later):
https://github.com/mono/mono/commit/796799ab55e8074cb33c1888a9a804cf251e42cf

File: mono/mini/simd-intrinsics-netcore.c [hunk 1]
```diff
@@ -258,10 +258,14 @@ emit_sys_numerics_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSigna
...
 		return ins;
 	case SN_ConvertToInt32:
+		if (!COMPILE_LLVM (cfg))
+			return NULL;
 		etype = get_vector_t_elem_type (fsig->params [0]);
 		g_assert (etype->type == MONO_TYPE_R4);
 		return emit_simd_ins (cfg, mono_class_from_mono_type_internal (fsig->ret), OP_CVTPS2DQ, args [0]->dreg, -1);
 	case SN_ConvertToSingle:
+		if (!COMPILE_LLVM (cfg))
+			return NULL;
```
Comment (filipnavara) [inline mono/mini/simd-intrinsics-netcore.c]: These ones (SN_ConvertToInt32, SN_ConvertToSingle) are actually implemented in mini-x64 already. Thanks for fixing the rest!

Comment (filipnavara) [reply mono/mini/simd-intrinsics-netcore.c]: Feel free to leave them disabled I will follow up on this and add the lowering code eventually.

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
The netcore failures are relevant.

**Pull request merged**